doc/man7/openssl-env.pod

```diff
@@ -95,6 +95,14 @@ Specifies a QUIC qlog output directory. See L<openssl-qlog(7)>.
 
 Used to set a QUIC qlog filter specification. See L<openssl-qlog(7)>.
 
+=item B<SSLKEYLOGFILE>
+
+Used to produce the standard format output file for SSL key logging.  Optionally
+set this variable to a filename to log all secrets produced by SSL connections.
+Note, use of the environment variable is predicated on configuring OpenSSL at
+build time with the enable-sslkeylog feature.  The file format standard can be
+found at L<https://datatracker.ietf.org/doc/draft-ietf-tls-keylogfile/>
```

Comment:
I'm wondering why we took the decision to have this "off" by default and configurable at build time.

Are there any security implications to having this?

Reply:
That was the reasoning yes, that we wanted builders of openssl to have to opt-in to doing this, so that they knew doing so created a risk in the possibility of an env variable injection that caused their secrets to get leaked